.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+For urgent operational issues, please contact AWS Support directly.
+https://aws.amazon.com/premiumsupport/
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!-- Not required for feature requests -->
+### Expected Behavior
+
+
+### Observed Behavior
+
+
+### Environment Details
+<!--
+Examples:
+* docker info
+* curl http://localhost:51678/v1/metadata
+* df -kh
+-->
+
+
+### Supporting Log Snippets
+<!--
+Please have a look at https://github.com/awslabs/ecs-logs-collector
+for the data we typically require for investigation
+
+GitHub issues are public, please remove sensitive data from logs before posting
```

Comment:
As per @nmeyerhans note above, `please remove sensitive data` is a very generic message. As a customer, it's very hard to parse Agent and other logs to determine if they contain any sensitive data. Should we provide more details here? Or should the guidance be that logs should be collected, but we'll provide a mechanism to share them with ECS team after the issue has been created?